geometry_msgs/PolygonStamped hdd_polygon
geometry_msgs/PolygonStamped pcb_polygon
---
int64 sum
AffordanceList computed_affordances 
